pybandits/base.py

```diff
@@ -201,3 +206,45 @@ def get_state(self) -> (str, dict):
         model_name = self.__class__.__name__
         state: dict = self.dict()
         return model_name, state
+
+    @validate_arguments
+    def _select_action(
```

Comment:
Let's call this function with an "epsilon-greedy" related name. E.g. `_select_action_epsilon_greedy`